lection1/cutString.js

```diff
@@ -0,0 +1,7 @@
+"use strict";
+
+function cutString(str, num) {
```

Comment:
use more declarative names for your arguments. `(text, availableChars)` looks rather cleaner